test/test_common/environment.cc

```diff
@@ -126,8 +126,10 @@ std::string TestEnvironment::substitute(const std::string& str,
   switch (version) {
   case Network::Address::IpVersion::v4:
     out_json_string = std::regex_replace(out_json_string, lookup_family_regex, "v4_only");
+    FALLTHRU;
```

Comment:
/me wasn't paying attention to the case, but just that it was implicitly falling through...